sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -316,10 +316,8 @@ export class ApplicationStudentsController extends BaseController {
     try {
       await this.applicationService.validateOverlappingDates(
         applicationId,
-        student.user.lastName,
         studentToken.userId,
-        student.sinValidation.sin,
-        student.birthDate,
+        student.id,
```

Comment:
The student ID is present in the `studentToken` also, using it can eliminate the call to the method `getStudentById` at line 303.